unofficial.gni

```diff
@@ -0,0 +1,286 @@
+    ]
+
+    sources = gypi_values.node_sources + [ "$target_gen_dir/node_javascript.cc" ]
+
+    if (is_win) {
+      libs = [ "psapi.lib" ]
+    }
+    if (is_mac) {
+      frameworks = [ "CoreFoundation.framework" ]
+    }
+
+    if (v8_enable_i18n_support) {
+      deps += [ "//third_party/icu" ]
+    }
+    if (node_use_openssl) {
+      public_deps += [ "deps/openssl" ]
+      sources += gypi_values.node_crypto_sources
+    }
+    if (node_enable_inspector) {
+      deps += [
+        "src/inspector:node_protocol_generated_sources",
+        "src/inspector:v8_inspector_compress_protocol_json",
+      ]
+      include_dirs = [
+        "$target_gen_dir",
+        "$target_gen_dir/src",
+        "$target_gen_dir/src/inspector",
+      ]
+      node_inspector = exec_script(
+          "./tools/gypi_to_gn.py",
+          [ rebase_path("src/inspector/node_inspector.gypi"),
+            "--replace=<(SHARED_INTERMEDIATE_DIR)=$target_gen_dir" ],
+          "scope",
+          [ "src/inspector/node_inspector.gypi" ])
+      sources += node_inspector.node_inspector_sources +
+                 node_inspector.node_inspector_generated_sources
+    }
+  }
+
+  executable(target_name) {
+    forward_variables_from(invoker, "*")
+
+    sources = [ "src/node_main.cc" ]
+    deps = [ ":libnode" ]
+    if (node_use_node_snapshot) {
+      sources += [ "$target_gen_dir/node_snapshot.cc" ]
+      deps += [ ":run_node_mksnapshot" ]
+      if (is_clang || !is_win) {
+        cflags_cc = [
+          "-Wno-c++11-narrowing",
+          "-Wno-shadow",
+        ]
+      }
+    } else {
+      sources += [ "src/node_snapshot_stub.cc" ]
+    }
+    output_name = "node"
+  }
+
+  if (node_use_node_snapshot) {
+    if (current_toolchain == v8_snapshot_toolchain) {
+      executable("node_mksnapshot") {
+        configs += [ ":node_internal_config" ]
+        sources = [
+          "src/node_snapshot_stub.cc",
+          "tools/snapshot/node_mksnapshot.cc",
+        ]
+        deps = [ ":libnode" ]
+      }
+    }
+
+    action("run_node_mksnapshot") {
+      deps = [ ":node_mksnapshot($v8_snapshot_toolchain)" ]
+      script = "$node_v8_path/tools/run.py"
+      sources = []
+      data = []
+
+      mksnapshot_dir = get_label_info(":node_mksnapshot($v8_snapshot_toolchain)",
+                                      "root_out_dir")
+
+      outputs = [ "$target_gen_dir/node_snapshot.cc" ]
+      args = [
+        "./" + rebase_path(mksnapshot_dir + "/node_mksnapshot", root_build_dir),
+        rebase_path("$target_gen_dir/node_snapshot.cc", root_build_dir),
+      ]
+    }
+  }
+
+  action("generate_config_gypi") {
+    script = "./tools/generate_config_gypi.py"
+    outputs = [ "$target_gen_dir/config.gypi" ]
+    depfile = "$target_gen_dir/$target_name.d"
+    script_args = [ "$root_build_dir" ]
+    script_args += outputs
+    script_args += [ depfile ]
+    args = rebase_path(script_args, root_build_dir)
+  }
+
+  action("node_js2c") {
+    script = "tools/js2c.py"
```

Comment:
I think this now needs to be updated to build and use the new `out/Release/node_js2c` binary.